src/lib/LoginFile.js

```diff
@@ -115,11 +115,11 @@ class LoginFile {
         /* eslint-disable no-multi-spaces */
         const $canvas = QrEncoder.render({
             text: encodedSecret,
-            radius: 0.7,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
+            radius: 0.5,    // We encode 56 bytes. To keep within a smaller QR base size, we need to reduce
```

Comment:
Comment still says `.7`.
That sentence and this line can just be removed. `.5` is already the default.